core/src/main/java/org/commonjava/maven/ext/core/impl/BaseGroovyManipulator.java

```diff
@@ -94,10 +94,12 @@
                 for ( final String script : scripts )
                 {
                     File found;
-                    if ( script.startsWith( "http" ) || script.startsWith( "file" ))
+                    logger.info( "Processing groovy script {}", script );
+                    if ( script.startsWith( "http:" ) || script.startsWith( "file:" ) )
                     {
-                        logger.info( "Attempting to read URL {} ", script );
-                        found = fileIO.resolveURL( new URL( script ) );
+                        URL url = new URL( script );
+                        logger.info( "Attempting to read URL {}", url );
+                        found = fileIO.resolveURL( url );
```

Comment:
This is the line that does not like the `groovyScripts=file://` URLs which gives `UnknownHostException: C` on Windows later when it does `url.openStream()` in `fileIO.resolveURL()`.

Reply:
I don't see why this is needed?

Reply:
I was trying to see if there was a difference between the given input URL which is `script` and the url after it has been parsed which is `url`. This was for debugging, so `logger.debug` could have been used.

Reply:
As this can convert the original `script` value I think it would be better kept as it was.

Reply:
There's no code that changed here. I just moved it to two separate lines so I could log the difference between the values. It looks like `URL.toString()` at the very least was printing only one slash in some cases, which is strange. In any case, I can revert it.